README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
+	- autoreconf (for bootstrapping)
```

Comment:
autoreconf is just a wrapper, so not really required and afais not used by `autogen.sh` either